Danki2/Assets/Prefabs/FireballObject.prefab

```diff
@@ -300,3 +301,19 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: ee701bb25599ede478b7c6c95e700704, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
+--- !u!54 &-5874146137788557195
+Rigidbody:
```

Comment:
I don't think the fireball object actually needs Rigidbody? As it doesn't need to do any physics, but just be a trigger. Thanks my understanding

Reply:
So for a collision to be registered, you need both components to have a collider and at least one to have a rigid body - not sure why - but we want it to collide with environment, and I think it makes more sense to have a rigidbody on the fireball than on the environment.

Reply:
Ah OK, yeah that sounds good